src/test/java/org/example/VerifyFilterAndSearchTest.java

```diff
@@ -0,0 +1,119 @@
+package org.example;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.List;
+
+public class VerifyFilterAndSearchTest {
+
+    @Test
+    public void verifyBrandName(){
+        System.setProperty(
+                "webdriver.chrome.driver",
+                "C:\\Users\\Asus\\Desktop\\Epam\\First_Selenium\\src\\test\\resources\\webdriver\\chromedriver.exe"
+        );
+        WebDriver webDriver = new ChromeDriver();
+        webDriver.get("https://www.amazon.com");
+        webDriver.manage().window().maximize();
```

Comment:
You can avoid code duplication by using separate method with annotation @BeforeMethod